wasm/hello-wasm/.nvmrc

```diff
@@ -0,0 +1 @@
+v18.12.1
```

Comment:
why we are restricting a fix version for this application???

Reply:
Well that's for the node version manager, so the developers that have automated script for node version switching can use this file, or someone who is not available download package can use this file to see the specific node version to use for the project. Though I don't know how it came here. I usually keep `.nvmrc` file in the node projects. It's probably a leftover.